.github/workflows/run_system_tests.yml

```diff
@@ -14,11 +14,30 @@ jobs:
     strategy:
       matrix:
         configuration: ["win-10-py32", "win-10-py64"]
+      # Fail-fast skews the pass/fail ratio and seems to make pytest produce
+      # incomplete JUnit XML results.
+      fail-fast: false
     timeout-minutes: 90
     steps:
       - name: Check out repo
         uses: actions/checkout@v3
       - name: Install dependencies
         run: poetry install
       - name: Run system tests
-        run: poetry run tox
\ No newline at end of file
+        run: poetry run tox
+      - name: Rename test results
+        # TODO: Add Git Bash to the path on self-hosted Windows runners
```

Comment:
Is this still relevant?

Reply:
Yes. If Git Bash was in the path, we wouldn't need a separate step that uses Powershell.

Git is installed, but only `%ProgramFiles%\Git\cmd` is in the path by default, not `%ProgramFiles%\Git\bin`

Reply:
[AB#2437386](https://dev.azure.com/ni/DevCentral/_workitems/edit/2375645/?workitem=2437386)